src/collection.rs

```diff
@@ -12,6 +20,8 @@ pub struct QueryParamsRaw {
     pub skip: Option<u64>,
     #[serde(rename = "$top")]
     pub top: Option<u64>,
+    #[serde(rename = "$filter")]
+    pub filter: Option<String>,
```

Comment:
does `odata_params` support `serde`?
if not we could implement a wrapper type that calls `odata_params::filters::parse_str()`
this way the parsing would be performed by `axum` and it would take care or returning correct errors.

Reply:
Hey @sergiimk

Unforgettably it doesn't support `serde`.